doc/findinfiles.rst

```diff
@@ -20,13 +19,15 @@ This will allow you to view and navigate through all the occurrences of your sea
 You can expand or collapse the search results to view the matches in each file.
 Clicking on a match will automatically open the file and highlight the line where the text was found.
 
-[GIF showing search and cliking file]
+.. image:: images/find_in_files/find-in-files-search.gif
```

Comment:
Pause at the end seems perhaps a second too long here?

Reply:
Agreed.

Reply:
Ok. Cutting 1 second at the end